DESCRIPTION

```diff
@@ -57,11 +57,11 @@ Collate:
     'CellGraphAssay.R'
     'aaa.R'
     'cell_count_plot.R'
+    'cell_rank_plot.R'
```

Comment:
```suggestion
    'molecule_rank_plot.R'
```

Reply:
I have updated the function name to `MoleculeRankPlot` now!

Reply:
Nice! I approve